src/index.html

```diff
@@ -11,6 +11,41 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
```

Comment:
Form styles on focus look strange. it doesn't match the design 
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/99403198/4c8112fc-1f3b-45ae-8e9e-8c13434d9d5f)